src/tab-controller/index.tsx

```diff
@@ -20,84 +21,136 @@ export enum Align {
 	top = 'top'
 }
 
+export interface TabItem {
+	closeable?: boolean;
+	disabled?: boolean;
+	id: string;
+	label: DNode;
+}
+
 export interface TabControllerProperties {
-	/** Position of the currently active tab */
-	activeIndex: number;
 	/** Orientation of the tab buttons */
 	alignButtons?: Align;
 	/** Custom aria attributes */
 	aria?: { [key: string]: string | null };
-	/** Called when a new tab button is clicked */
-	onRequestTabChange?(index: number, key: string): void;
-	/** Called when a tab close button is clicked */
-	onRequestTabClose?(index: number, key: string): void;
+	/** Index of the initial active tab. Defaults to 0. */
+	initialIndex?: number;
```

Comment:
Could this be the tab id rather than index?

Reply:
Yes. Good catch.